cmd/dbfactory/cluster.go

```diff
@@ -79,3 +93,78 @@ func printJSON(data interface{}) error {
 	encoder.SetIndent("", "    ")
 	return encoder.Encode(data)
 }
+
+// searchOpts the options to be used for the search command
+type searchOpts struct {
+	tags   map[string]string
+	engine string
+	limit  int64
+}
+
+// newSearchCommand adds a subcommand in cluster in order to be
+// able to search by given tags.
+// - Example -
+// dbfactory cluster -t 'DatabaseType=multitenant-rds'
+func newSearchCommand(svc rdsiface.RDSAPI) *cobra.Command {
+	opts := searchOpts{}
+
+	cmd := &cobra.Command{
+		Use:   "get",
+		Short: "Returns the RDS clusters which has been created by database factory server.",
+		RunE: func(command *cobra.Command, args []string) error {
+			input := &rds.DescribeDBClustersInput{
+				MaxRecords: aws.Int64(opts.limit),
+			}
+
+			result, err := svc.DescribeDBClusters(input)
+			if err != nil {
+				return err
+			}
+			table := tablewriter.NewWriter(os.Stdout)
+			table.SetHeader([]string{"VPC", "Database ID", "Engine", "Engine Version", "Backup Retention"})
+
+			for _, r := range result.DBClusters {
+				group, err := svc.DescribeDBSubnetGroups(&rds.DescribeDBSubnetGroupsInput{
+					DBSubnetGroupName: r.DBSubnetGroup,
+				})
+				if err != nil {
+					fmt.Printf("Failed to DBSubnetGroup: %s \n\n", *r.DBSubnetGroup)
+					return err
```

Comment:
nit: I think we could wrap the error here and rely on cobra to print it:
```suggestion
					return errors.Wrapf(err, "failed to describe DBSubnetGroup: %s", *r.DBSubnetGroup)
```